### PR TITLE
Feature/add city info data

### LINKED ISSUE #19: Consume data from Backend Endpoint
- [ ] Currently we are using mockData to display the information on the DOM. We need to convert to fetching the information from the backend's endpoint

### PR DESCRIPTION
**Pull request by kendallm360**
## Summary:
Added quality scores and housing data to the CityInfo component. Also added initial styling to the page. Installed and implemented progress bars

## Type of Changes:
- [ ] Bug fix
- [ ] Refactor
- [x] New feature

## What are the relevant tickets:
closes #19

### CODE REVIEW

File: src/components/CityInfo/CityInfo.js [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
   return (
-    <>
+    <StyledCityInfo>
       <div className="city-info-container">
         <div className="city-name">
-          <p>I'm a freaking CityInfo component!!!!</p>
-          <p>{city.attributes.name}</p>
+          <h2>{city.attributes.name}</h2>
+          <img src={city.attributes.details.image_mobile_url} />
+          <Quality city={city} />
```
Comment (BrandonAinsworth) [inline src/components/CityInfo/CityInfo.js:20]: Nice job DRYing up this component by adding, more components!

File: src/components/Progress/Progress.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { ProgressBar } from "react-bootstrap";
+import { StyledProgress } from "../styles/Progress.styled";
+const Progress = ({ score, city, name }) => {
+  return (
+    <StyledProgress>
```
Comment (BrandonAinsworth) [inline src/components/Progress/Progress.js:6]: Great work on the progress bar.

File: src/components/styles/Housing.styled.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+export const StyledHousing = styled.section`
```
Comment (BrandonAinsworth) [inline src/components/styles/Housing.styled.js:3]: Nice use of styled components throughout

File: src/components/Housing/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import CityCard from "../CityCard/CityCard";
```
Comment (BrandonAinsworth) [inline src/components/Housing/index.js:1]: Using index.js for each component really shows your maturity as a developer.

**Review by BrandonAinsworth — APPROVED** [review]
Awesome job fleshing out CityInfo

**Comment by omegaeye-he** [comment]
Nice comments and commits. I don't see any tests, so how do we know if it's all working and there are no failures?

**Pull request merged**